packages/shorebird_cli/lib/src/commands/build/build_apk_command.dart

```diff
@@ -23,7 +23,18 @@ class BuildApkCommand extends ShorebirdCommand
     required super.logger,
     super.auth,
     super.validators,
-  });
+  }) {
+    argParser
+      ..addOption(
+        'target',
+        abbr: 't',
+        help: 'The main entrypoint file of the application.',
+      )
+      ..addOption(
+        'flavor',
+        help: 'The product flavor to use when building the app.',
```

Comment:
It'd be helpful to give examples of flavors or otherwise hint how a developer might use them, either here or in the readme.

Reply:
good call I'll update the README in one go in the next PR